src/filters/FilterSystem.ts

```diff
@@ -201,19 +200,16 @@ export class FilterSystem implements System
         {
             const filter = filters[i];
 
-            resolution = Math.min(resolution, filter.resolution);
+            resolution = Math.min(resolution, filter.resolution ?? colorTextureSource._resolution);
             padding += filter.padding;
 
-            if (filter.antialias !== 'inherit')
+            if (filter.antialias === 'on')
             {
-                if (filter.antialias === 'on')
-                {
-                    antialias = true;
-                }
-                else
-                {
-                    antialias = false;
-                }
+                antialias = true;
+            }
+            else if (filter.antialias === 'inherit')
+            {
+                antialias ||= colorTextureSource.antialias;
```

Comment:
The comment of `antialias` says:
> if this is true for any filter, it should be true

The previous logic didn't match the comment. Fixed that.

Reply:
Though it is a bit odd that we use the lowest resolution, but antialiasing if any of the filters has it. I feel like it should be either lowest resolution + antialias if all filters have antialia,s or highest resolution + antialias if any of them has it. In V7 we chose the lowest multisample count.

Reply:
good catch - you are correct - should be true only if all filters have antialiasing!

Reply:
Fixed.